scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -340,6 +340,7 @@ VAL_ 300 SECONDARY_FLOW_RATE_OUT_OF_RANGE 0 "OK" 1 "UNDERFLOW" 2 "OVERFLOW";
 VAL_ 300 STEERING_ANGLE_OUT_OF_RANGE 0 "OK" 1 "UNDERFLOW" 2 "OVERFLOW";
 VAL_ 300 BRAKE_PRESSURE_OUT_OF_RANGE 0 "OK" 1 "UNDERFLOW" 2 "OVERFLOW";
 VAL_ 305 State 0 "AIR_OPEN" 1 "AIR_CLOSED";
+VAL_ 308 Brake_Is_Actuated 0 "INACTIVE" 1 "ACTIVE";
```

Comment:
@thekenu  Was thinking if I should rename the constants to "FALSE" and "TRUE" here?

Reply:
Yeah I don't mind